### DISCUSSION TITLE
Sell/Grab command is glitching (2nd time)

### DISCUSSION THREAD

**Discussion by Dexteriority** [winstxnhdw/lc-hax#363] [Q&A] [answered]
The same as the old issue we ever talked about in the previous discussion. Is it just me or?..
The Player Weight messed up after using the /grab or /sell command.

/grab = became faster like The Flash 
/sell = became slower than turtle

**Comment by winstxnhdw** [comment, marked as answer]
I have fixed the problem with `grab` in latest, but I never really changed anything in `sell` so I am not sure why the issue is recurring.

**Reply by Dexteriority** [reply]
I will look for it again and will tell you if i encounter another glitch
